src/components/main-body/MainBody.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import cn from 'classnames';
+
+import styles from './MainBody.scss';
+
+const MainBody = ({ children }) => <div className={cn('row', styles.mainBody)}>{children}</div>;
```

Comment:
is `row` required here? as bootstrap was already removed. Or is this a custom class